runtime/calamari/tests/xcm_mock/xcm_tests.rs

```diff
@@ -350,6 +352,90 @@ fn send_para_a_native_asset_to_para_b() {
     ParaB::execute_with(|| {
         // free execution, full amount received
         assert_eq!(parachain::Assets::balance(a_asset_id_on_b, &ALICE), amount);
+        assert_eq!(
+            parachain::Assets::balance(a_asset_id_on_b, &XcmFeesAccount::get()),
+            0
+        );
+    });
+}
+
+#[test]
+fn send_para_b_asset_to_para_b() {
+    MockNet::reset();
+
+    let para_a_source_location = create_asset_location(1, PARA_A_ID);
+    let para_b_source_location = create_asset_location(1, PARA_B_ID);
+
+    let amount = INITIAL_BALANCE;
+
+    let para_a_asset_metadata = create_asset_metadata("ParaAToken", "ParaA", 18, 1, false, false);
+    let para_b_asset_metadata = create_asset_metadata("ParaBToken", "ParaB", 18, 1, false, false);
+
+    let _ = register_assets_on_parachain::<ParaA>(
+        &para_a_source_location,
+        &para_a_asset_metadata,
+        Some(0u128),
+        None,
+    );
+    let b_asset_id_on_a = register_assets_on_parachain::<ParaA>(
+        &para_b_source_location,
+        &para_b_asset_metadata,
+        Some(0u128),
+        None,
+    );
+
+    let _ = register_assets_on_parachain::<ParaB>(
+        &para_b_source_location,
+        &para_b_asset_metadata,
+        Some(0u128),
+        None,
+    );
+    let _ = register_assets_on_parachain::<ParaB>(
+        &para_a_source_location,
+        &para_a_asset_metadata,
+        Some(0u128),
+        None,
+    );
+
+    let dest = MultiLocation {
+        parents: 1,
+        interior: X2(
+            Parachain(PARA_B_ID),
+            AccountId32 {
+                network: NetworkId::Any,
+                id: ALICE.into(),
+            },
+        ),
+    };
+
+    // Transfer ParaB balance to B
+    // Also tests that a sender can send all of their balance
+    ParaA::execute_with(|| {
+        assert_ok!(parachain::Assets::mint_into(
+            b_asset_id_on_a,
+            &ALICE,
+            amount
+        ));
+        assert_eq!(parachain::Assets::balance(b_asset_id_on_a, &ALICE), amount);
+
+        assert_ok!(parachain::XTokens::transfer(
+            parachain::Origin::signed(ALICE),
+            parachain::CurrencyId::MantaCurrency(b_asset_id_on_a),
+            amount,
+            Box::new(VersionedMultiLocation::V1(dest)),
+            ADVERTISED_DEST_WEIGHT
+        ));
+        assert_eq!(
+            parachain::Assets::balance(b_asset_id_on_a, &ALICE),
+            INITIAL_BALANCE - amount
```

Comment:
This is just `INITIAL_BALANCE` -`INITIAL_BALANCE`, maybe just make zero?

Or perhaps the above line `let amount = INITIAL_BALANCE` should be a number like in the other tests

Reply:
change to value 0, and in `send_para_b_asset_to_para_b_with_trader_and_fee` testcase, set amount to other value.